src/sentry/api/serializers/rest_framework/dashboard.py

```diff
@@ -863,6 +878,14 @@ def create_widget(self, dashboard, widget_data):
 
         DashboardWidgetQuery.objects.bulk_create(new_queries)
 
+        # Handle field links for each query
+        for i, query_obj in enumerate(new_queries):
+            query_data = query_data_list[i]
+            if "linked_dashboards" in query_data and query_data["linked_dashboards"]:
```

Comment:
Could walrus this maybe, `if linked_dashboards := query_data.get("linked_dashboards", []):`